atlasdb-workload-server-antithesis/var/docker-compose.yml

```diff
@@ -20,7 +20,7 @@ services:
         ipv4_address: 10.20.20.2
 
   cassandra2:
-    image: palantirtechnologies/cassandra:2.2.18-1.112.0-rc5
+    image: palantirtechnologies/cassandra:2.2.18-1.112.0-rc8
```

Comment:
And bumps to the latest C* RC, which are nice to have but not critical for this PR. Happy to split if reviewer deems necessary.